entities.py

```diff
@@ -0,0 +1,118 @@
+from dices import Dices
+import random
+
+
+class Movement():
+    def __init__(self):
+        self.position = [0, 0]  # top left corner as base start pos
+        self.board_size = 4  # Boardsize set to 4 as base value
+        self.options = []
+        self.check_options()
+
+    def check_options(self):
+        option = []
+        # y
+        if self.position[0] + 1 <= self.board_size - 1:  # position 0 indexed
+            option.append('South')
+        if self.position[0] - 1 >= 0:
+            option.append('North')
+        # x
+        if self.position[1] + 1 <= self.board_size - 1:
+            option.append('East')
+        if self.position[1] - 1 >= 0:
+            option.append('West')
+        self.options = option
+
+    def move(self, axle, direction):
+        # Axle x or y (0 or 1), direction = 1 or -1
+        self.position[axle] += direction
+
+
+class Entity(Movement):
+    # Contains character funcs such as attacking, aswell as holds the movement
+    def __init__(self):
+        super().__init__()
+
+    def attack_roll(self):
+        # return attack roll
+        return Dices.d6(self.attack)
+
+    def evade_roll(self):
+        # return evasion roll
+        return Dices.d6(self.agility)
+
+    def take_hit(self):
+        # reduce HP by 1
+        self.endurance -= 1
+
+    def initiative_roll(self):
+        # roll for initiative
+        return Dices.d6(self.initiativ)
+
+    def escape_roll(self):
+        # Returns bool for escape attemt
+        escape = False
+        if random.randint(1, 10) <= self.agility:
+            escape = True
+        return escape
+
+
+# Hero classes
+class Rider(Entity):
+    def __init__(self):
+        super().__init__()
+        self.initiativ = 5
+        self.endurance = 9
+        self.attack = 6
+        self.agility = 4
+
+
+class Wizard(Entity):
```

Comment:
We could redefine the 'escape_roll' for the wizard in the characters subclass, as it will always have an 80% chance to escape